direct/src/dist/commands.py

```diff
@@ -401,14 +401,22 @@ def download_wheels(self, platform):
 
         whldir = os.path.join(self.build_base, '__whl_cache__')
 
+        pip_version = [int(i) for i in pip.__version__.split('.')][0]
```

Comment:
I'm not sure why you changed this, since we only are grabbing the first component.

It's legal for a version number to contain letters in the final components, so I worry that this might fail for a release candidate.

Reply:
I first kept pip_version as an array and then changed it to just grab the major version. I agree that this could cause issues with letters in version names and I'll clean it up.